run.sh

```diff
@@ -25,3 +25,4 @@ run "PHP" "php ./php/code.php 40"
 run "R" "Rscript ./r/code.R 40"
 run "Python" "python3 ./py/code.py 40" 
 run "Dart" "./dart/code 40"
+run "C#" "./csharp/bin/Release/net9.0/linux-x64/native/code 40"
```

Comment:
Is it possible to configure path to be generic? Otherwise it doesn't seem to be very portable to have version & platform hardcoded in the path

Reply:
Ah yes. I had an idea around that but the best idea I had was to introduce cli args to the bash. And still, I am not sure adding this extra feature would be a good idea because of how the script was intended (i.e. to be executed as it is).

I am open to any idea but IMO it is good enough the way it is.

Reply:
You can do it just by adding the `-o` argument to the  `dotnet publish` command and setting it to a `/bin` path instead of the `.NET version + arch` path.

Reply:
Seems perfect.

Reply:
The only missing part is the target arch, it is still linux-x64. But that should ok for 99% of devs outside Windows I suppose.

Reply:
Now it is agnostic. Perfect!